playbooks/azure/openshift-cluster/build_node_image.yml

```diff
@@ -94,6 +94,19 @@
     with_items:
     - { regexp: '^ResourceDisk\.Format=', line: 'ResourceDisk.Format=n' }
 
+  # set waagent to start after docker
+  - name: set waagent systemd file
+    copy:
+      dest: /usr/lib/systemd/system/waagent.service
```

Comment:
override should go in /etc rather than patching /usr - check the systemd docs

Reply:
see /etc/systemd/system/docker.service.d/custom.conf on our node image as an example of how to do what I mean

Reply:
Updated the file to place the changes in /etc

Reply:
Personally I feel the .d directory is overkill, having `/etc/systemd/system/waagent.service` would've been enough :man_shrugging: